tests/Database/DatabaseEloquentBelongsToManyTest.php

```diff
@@ -554,6 +554,50 @@ public function testTouchMethodSyncsTimestamps()
         $relation->touch();
     }
 
+    /**
+     * @dataProvider toggleMethodListProvider
+     */
+    public function testToggleMethodTogglesIntermediateTableWithGivenArray($list)
+    {
+        $relation = $this->getMockBuilder('Illuminate\Database\Eloquent\Relations\BelongsToMany')->setMethods(['attach', 'detach'])->setConstructorArgs($this->getRelationArguments())->getMock();
```

Comment:
Hi @JosephSilber,
I made the tests based on the existing sync's ones. But if it's necessary I can update them.
But, in that case, maybe the "sync" method tests also need an update, do you think?